Docs: Code contribution WSL removal (#29174)
storybookjs/storybook: docs/contribute/code.mdx

## Prerequisites

* Ensure you have Node version 18 installed (suggestion: v18.16.0).

> * Ensure if you are using Windows to run the commands below in a terminal with administrator privileges.

Reviewer: ```suggestion
* If you're working with Windows, all commands should be run in a terminal with administrator privileges.
```
